blockchain/tx_pool_test.go

```diff
@@ -309,19 +320,21 @@ func TestInvalidTransactions(t *testing.T) {
 	pool.gasPrice = big.NewInt(1000)
 
 	// NOTE-Klaytn We only accept txs with an expected gas price only
-	//         regardless of local or remote.
-	// TODO : Need to KIP-71 hardfork.
-	//if err := pool.AddRemote(tx); err != ErrInvalidUnitPrice {
-	//	t.Error("expected", ErrInvalidUnitPrice, "got", err)
-	//}
-	//if err := pool.AddLocal(tx); err != ErrInvalidUnitPrice {
-	//	t.Error("expected", ErrInvalidUnitPrice, "got", err)
-	//}
-	if err := pool.AddRemote(tx); err != ErrGasPriceBelowBaseFee {
-		t.Error("expected", ErrGasPriceBelowBaseFee, "got", err)
-	}
-	if err := pool.AddLocal(tx); err != ErrGasPriceBelowBaseFee {
-		t.Error("expected", ErrGasPriceBelowBaseFee, "got", err)
+	// regardless of local or remote.
+	if pool.chainconfig.IsKIP71ForkEnabled(pool.chain.CurrentBlock().Number()) {
+		if err := pool.AddRemote(tx); err != ErrGasPriceBelowBaseFee {
+			t.Error("expected", ErrGasPriceBelowBaseFee, "got", err)
+		}
+		if err := pool.AddLocal(tx); err != ErrGasPriceBelowBaseFee {
+			t.Error("expected", ErrGasPriceBelowBaseFee, "got", err)
+		}
+	} else {
```

Comment:
I don't think this else logic will be operated.
How about set kip71 value of the tx pool like `pool.kip71 = true/false` and then run `AddRemote` and `AddLocal`?

Reply:
Right, we can leave this code for before kip71 hardfork because `TestTransactionNotAcceptedWithLowerGasPrice` can cover this situation.